examples/save_config_demo/data/main.js

```diff
@@ -51,6 +52,14 @@ $( document ).ready(function() {
         o[this.name] = value || '';
       }
     });
+
+    var c = $('input[type=radio],input[type=checkbox]',this);
+    $.each(c, function(){
+      if(!o.hasOwnProperty(this.name)){
+        o[this.name] = false;
+      }
+    });
```

Comment:
someones been busy in the new year!! ;)

my JS is very rusty .. this Looks good, the only thing I'd wondering is, are these not already iterated on via
```
var a = this.serializeArray();
$.each(a, function()
```

Reply:
Had a little free time :) 

I had originally thought so, but alas not. It only comes up there if it is checked. This is why the check if it already exists.

Reply:
oh, I remember this sort of now, since its 'empty' the browser treats it as invisible or something. I _think_ there is a way to iterate everything on one attempt, but def. not worth the cycles for the demo IMO ... 👍